tests/test_redmine.py

```diff
@@ -41,40 +43,91 @@ def fail_line_matrix():
     return lines
 
 
+multiple_ticket_lines = [
+    'issues #1 #2 #3',
```

Comment:
Yeah, this test is not as nice as your matrix ones - it was over my head to set up the full matrix like you've got above, there.